### PR TITLE
extend file metadata with creation date

### PR DESCRIPTION
**Pull request by samsamann**
I added a new metadata attribute for the file upload service. The creation date attribute is set when uploading a file.

### CODE REVIEW

**Comment by hairmare** [comment]
The travis-ci fail will be fixed if you merge develop into your branch... You'll also want to add some stuff to ``src/Graviton/FileBundle/Tests/Controller/FileControllerTest.php`` (specially for the read-only part).

**Comment by narcoticfresh** [comment]
@hairmare This should be ready now.. I added the `readOnly` tests and renamed the awfully named field `cTime` :walking:

File: src/Graviton/FileBundle/Controller/FileController.php [hunk 1]
```diff
@@ -92,11 +92,14 @@ public function postAction(Request $request)
         $request->attributes->set('id', $record->getId());
 
         $file = $this->saveFile($record->getId(), $request->getContent());
+
         // update record with file metadata
         $meta = new FileMetadata();
         $meta->setSize((int) $file->getSize())
-            ->setMime($request->headers->get('Content-Type'));
+            ->setMime($request->headers->get('Content-Type'))
+            ->setCreatedate(new \DateTime());
```
Comment (hairmare) [inline src/Graviton/FileBundle/Controller/FileController.php]: Shouldn't this be ``setCreateDate``?

Comment (narcoticfresh) [inline src/Graviton/FileBundle/Controller/FileController.php]: No, [g|s]etters are generated `ucfirst(strtolower())`..

Comment (hairmare) [inline src/Graviton/FileBundle/Controller/FileController.php]: aight...

**Pull request merged**